client/src/main/webjar/organization-form/organization-form.html

```diff
@@ -59,6 +68,9 @@
             <input name="organizationFacebook"
                    ng-model="organizationForm.organization.facebookUrl"
                    type="url">
+            <div ng-messages="organizationForm.orgForm.organizationFacebook.$error">
+                <div ng-message="url">Organization Facebook page must be a valid url</div>
```

Comment:
We can come back to these later, but maybe we should mention what a valid URL looks like "start with https://" or something.